terraform/per_account/deployable/cloudwatch_alarm__events__TriggeredRules.tf

```
@@ -0,0 +1,52 @@
+variable "eu-west-1__events__TriggeredRules" {
+  description = "List of metrics derived from aws cloudwatch list-metrics"
+  type        = list
+  default     = []
+}
+
+variable "eu-west-2__events__TriggeredRules" {
+  description = "List of metrics derived from aws cloudwatch list-metrics"
+  type        = list
+  default     = []
+}
+
+resource "aws_cloudwatch_metric_alarm" "euw1_cloudwatch_events_triggered_rules" {
+  # iterate over count to setup multiple alarms
+  count               = length(var.eu-west-1__events__TriggeredRules)
+  provider            = aws.eu-west-1
+  alarm_name          = "${var.eu-west-1__events__TriggeredRules[count.index].ResourceName}_TriggeredRules_alarm"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = 1
+  threshold           = var.eu-west-1__events__TriggeredRules[count.index].Threshold
+  alarm_description   = "Check the number of messages added to the queue is not too high."
+  metric_name         = "TriggeredRules"
+  namespace           = "AWS/Events"
+  period              = 300
+  statistic           = "Maximum"
+  dimensions = {
+    RuleName = var.eu-west-1__events__TriggeredRules[count.index].ResourceName
+  }
+  alarm_actions       = ["${local.euw1_sns_cloudwatch_forwarder_topic}"]
+  ok_actions          = ["${local.euw1_sns_cloudwatch_forwarder_topic}"]
```

Do these need to use string interpolation?
```suggestion
  alarm_actions       = [local.euw1_sns_cloudwatch_forwarder_topic]
  ok_actions          = [local.euw1_sns_cloudwatch_forwarder_topic]
```

---

Due to the prevalence of this through out the terraform, I'll fix this in another PR.